packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -137,6 +139,19 @@ class EksClusterStack extends TestStack {
     // apply a kubernetes manifest
     this.cluster.addManifest('HelloApp', ...hello.resources);
   }
+  private assertManifestWithoutValidation() {
+    // apply a kubernetes manifest
+    new eks.KubernetesManifest(this, 'HelloAppWithoutValidation', {
+      cluster: this.cluster,
+      manifest: [{
+        apiVersion: 'v1',
+        kind: 'ConfigMap',
+        data: { hello: 'world' },
+        metadata: { name: 'config-map' },
+      }],
+      skipValidation: true,
```

Comment:
Im wondering if we can have an illegal manifest here that would fail if this property isn't satisfied, that would actually test the expected behavior. Maybe just mangle the config map?

Reply:
Thanks for this! I was able to notice a couple of bugs because of it:
1. To treat the string-boolean passed from provider correctly ("true"/"false"). Fixed this even for `overwrite`.
2. Include the validation as part of `create` event as well as `update`

Reply:
When I say I fixed it even for overwrite, I mean to say that previous implementation would have treated overwrite as truthy since "false" string would have been considered truthy by Python.

Reply:
👍